src/game/gun/gun_rifle.c

```diff
@@ -1,35 +1,31 @@
-#include "game/lara.h"
+#include "game/gun/gun_rifle.h"
 
 #include "config.h"
+#include "game/gun/gun_misc.h"
 #include "game/input.h"
 #include "game/random.h"
 #include "game/sound.h"
-#include "global/const.h"
-#include "global/types.h"
 #include "global/vars.h"
 
-#include <stddef.h>
-#include <stdint.h>
-
-void DrawShotgun(void)
+void Gun_Rifle_Draw(void)
```

Comment:
Because it's a family of functions related to drawing two-handed weapons. TR1 only happens to have Shotgun, but TR2 introduces M16 and TR3 harpoon and grenade launcher.

Similarly, pistols is for the default pistols, magnums and uzis.

I just made it forwards compatible with the naming in T3M.

Reply:
Ok that makes sense.

Reply:
Thinking about this more, what about naming it something like two handed weapon class (shortened somehow). I still think it's kind of confusing that shotgun is in rifle class. Maybe the original naming scheme was indicating LaraGun1 = one gun and LaraGun2 = dual wield.

Reply:
For whatever my opinion counts here, I must agree that calling a shotgun a rifle seems confusing.

Reply:
@walkawayy I was about to propose `1H` and `2H` but these are still confusing because what does `2H` mean? Two-handed? Dual wield?

Reply:
Gun_Draw_2Handed and Gun_Draw_1Handed?

Reply:
Since `2handed` and `1handed` is the more generic prefix, `Gun_1Handed_Draw` `Gun_2Handed_Draw`?

Reply:
Ya that sounds good.

Reply:
Actually I don't like this.
I look at "2 handed" and immediately think of pistols (because of dual wield) and only later of a shotgun (because you need both hands to handle it). Likewise "1 handed" refers to pistols, but Lara wears two of them, so "1" is confusing to me.

Rifle and pistols do not have such dilemmas.

Reply:
`Gun__Draw_Heavy` and `Gun_Draw_Pistol`? `Gun_Draw_` and `Draw_Gun_Pistol`? Idk what else to call it. Seems like you have your mind set, so I guess just leave it. It matches TR2 it looks like:
https://github.com/Arsunt/TR2Main/blob/5bec0d77667276d37eef4e4e23d295a06ae2a596/game/lara1gun.cpp#L45

Reply:
Yeah it started to require the weapon_type with the introduction of the M16, Harpoon and the Grenade Launcher.